### PR TITLE
Adding ability to configure EC2 disk size

### PR DESCRIPTION
**Pull request by ChrisFriesen**

### CODE REVIEW

File: src/bin/nitrogen.rs [hunk 1]
```diff
@@ -40,6 +40,9 @@ enum Commands {
         /// EC2 instance type. Must be Nitro Enclaves compatible
         #[arg(long, default_value_t = String::from("m5a.xlarge"))]
         instance_type: String,
+        /// EC2 root disk size
```
Comment (jvmncs) [inline src/bin/nitrogen.rs:43]: what's the units? would be good to add here

File: src/bin/nitrogen.rs [hunk 2]
```diff
@@ -106,6 +110,9 @@ enum Commands {
...
```
Comment (jvmncs) [inline src/bin/nitrogen.rs:113]: missing units, as above

File: src/bin/nitrogen.rs (hunk 1 above)
Comment (jvmncs) [reply src/bin/nitrogen.rs:43]: same w/ the implicit max of `16384`

File: src/bin/nitrogen.rs (hunk 2 above)
Comment (jvmncs) [reply src/bin/nitrogen.rs:113]: same w/ the implicit max of `16384`

**Review by jvmncs — APPROVED** [review]

**Comment by jvmncs** [comment]
looks like CI is failing from clippy for a new rust version

File: src/bin/nitrogen.rs (hunk 1 above)
Comment (ChrisFriesen) [reply src/bin/nitrogen.rs]: Good point, added GiBs to the base description here and a constraint description to the template

**Pull request merged**